src/appservice/AppService.ts

```diff
@@ -257,12 +272,28 @@ export class MjolnirAppService {
           result.error
         );
       }
+      // Send a notice that the invite must be accepted
+      await this.bridge
+        .getBot()
+        .getClient()
+        .sendText(
+          mxEvent.room_id,
+          "Please accept the invites to the newly provisioned rooms. These will be the home of your Draupnir Instance. This room will not be used in the future."
+        );
     } catch (e: unknown) {
       log.error(
         `Failed to provision a mjolnir for ${mxEvent.sender} after they invited ${this.bridge.botUserId}:`,
         e
       );
       // continue, we still want to reject this invitation.
+      // Send a notice that the provisioning failed
+      await this.bridge
+        .getBot()
+        .getClient()
+        .sendText(
+          mxEvent.room_id,
+          "Please make sure you are allowed to provision a bot. Otherwise notify the admin please. The provisioning request was rejected."
```

Comment:
```suggestion
          "Please make sure you are allowed to provision a bot. Otherwise please notify the admin. The provisioning request was rejected."
```